src/v/raft/tests/persisted_stm_test.cc

```diff
@@ -438,7 +446,7 @@ struct persisted_stm_test_fixture : state_machine_fixture {
                     return batches.back().last_offset();
                 });
           });
-
+        fmt::print(">>>> snapshot offset: {}\n", snapshot_offset);
```

Comment:
leftover or we vlog is not liked int he raft tests?

Reply:
leftover